changelog/1040.feature.rst

```diff
@@ -0,0 +1 @@
+Support ``animated`` emoji field in reaction removal events (e.g. :func:`on_raw_reaction_remove`).
```

Comment:
versionchanged on the event documentation?

Reply:
sounds good.
https://github.com/DisnakeDev/disnake/pull/1040/commits/278cb44ff023884ab63b11a8330ba3af9eadb984